### DISCUSSION TITLE
Does it work on raspberry pi running raspbian?

### DISCUSSION THREAD

**Discussion by yeahman45** [tjko/nxgipd#17] [General]
Does it work on raspberry pi running raspbian?

**Comment by Orgjvr** [comment]
Yes, I am running it on my Raspbian based pi.

pi@oh:~/Projects/nxgipd $ cat /etc/os-release 
PRETTY_NAME="Raspbian GNU/Linux 9 (stretch)"
NAME="Raspbian GNU/Linux"
VERSION_ID="9"
VERSION="9 (stretch)"
